src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,71 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = getNumbers(1, Math.ceil(total / perPage)).map(n => n);
```

Comment:
The '.map(n => n)' portion of this line is redundant, as it simply returns the same array. You can remove this part of the code.